{"levelname": "ERROR", "asctime": "2024-01-25T05:33:04.025Z", "name": "processing", "message": "Unable to publish collection version.", "lineno": 69, "pathname": "/single-cell-data-portal/backend/layers/processing/publish_revisions.py", "collection_id": "d180e6d5-544b-4fb2-8616-f4faa3d8f8a5", "collection_version_id": "06461f0a-eb2a-433a-8413-f108bf1830be", "errors": ["error message"]}
